html/rendering/widgets/input-checkbox-switch-notref.html

```diff
@@ -0,0 +1,4 @@
+<!doctype html>
+<input type=checkbox>
+<input type=checkbox>
+<input type=checkbox>
```

Comment:
```suggestion
<input type=checkbox>
<input type=checkbox>
<input type=checkbox switch>
```
The third one can have the switch attribute, since the test file doesn't have it on the third one.